Casamari/MNC008.xml

```diff
@@ -135,85 +276,220 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                           original sewing threads are visible in the centre fold of <ref
+                              target="#q5"/> (<locus target="#39v"/>). Some leaves received a
+                           parchment repair on the upper edge (<locus target="#22"/>, <locus
+                              from="61" to="64"/>). All edges were trimmed, some characters cut
+                           (e.g. <locus target="#5r #8r"/>).</condition>
                      </supportDesc>
+
                      <layoutDesc>
-                        <layout columns="1" writtenLines="12 15">
-                           <dimensions>
+                        <layout columns="1" writtenLines="12 15">The number of written lines varies
+                           from 12 to 15 throughout the manuscript. <locus from="3r" to="64v"/>
+                           <note corresp="#textarea #margin">Data on text area and margin dimensions
+                              taken from <locus target="#8r"/>.</note>
+                           <dimensions unit="mm" xml:id="textarea">
                               <height>98</height>
-                              <width>65</width>
+                              <width>60</width>
                            </dimensions>
-                           <dimensions type="margin">
+                           <dimensions type="margin" unit="mm" xml:id="margin">
                               <dim type="top">3</dim>
                               <dim type="bottom">20</dim>
                               <dim type="left">5</dim>
                               <dim type="right">13</dim>
                            </dimensions>
-                           <note>
-                              Data on text area dimensions 
-                              taken from <locus target="#8r"/>.
-                           </note>
+                           <note>The characters per line are 9–13.</note>
+                           <ab type="pricking">Ruling is visible; pricking is visible.</ab>
+                           <ab type="ruling" subtype="pattern">
+                              <!--TO BE CHECKED BY MARCO:-->
+                              <locus from="4r" to="41v"/>Ruling pattern: 1A-1A/0-0/0-0/C.</ab>
+                           <ab type="pricking">Primary pricks are visible.</ab>
+                           <ab type="pricking">Ruling pricks are mostly visible.</ab>
+                           <ab type="ruling">The upper line is written above the ruling.</ab>
+                           <ab type="ruling">The bottom line is written above the ruling.</ab>
+                           <ab type="ruling"><locus from="1r" to="2v"/>, <locus target="#40v #44v"
+                              />, <locus from="62v" to="63r"/> are not ruled.</ab>
                            <ab type="pricking">Pricking and ruling are visible.</ab>
+                           <ab type="ruling">A very few folia have a top marginal line as on <locus
```

Comment:
I would replace "as on" with "e.g." (the subject of the sentence is the folia, so "on" seems redundant to me)